### PR TITLE
Add GoogleCloudPlatform/compute-image-tools test results to testgrid.

### PR DESCRIPTION
**Pull request by crunk1**

### CODE REVIEW

**Comment by BenTheElder** [comment]
/ok-to-test
maybe use an external link in the PR?

**Comment by krzyzacy** [comment]
I think you want to add the bucket to https://github.com/kubernetes/test-infra/blob/master/buckets.yaml as well

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -1937,6 +1940,11 @@ dashboards:
   - name: gce-min-node-permissions
     test_group_name: ci-kubernetes-e2e-gce-min-node-permissions
 
+- name: google-gce-compute-image-tools
+  dashboard_tab:
+    - name: ci-daisy-e2e
```
Comment (michelle192837) [inline testgrid/config/config.yaml]: Would do 'daisy-e2e' or 'daisy' to be more in line with other names.

Comment (crunk1) [reply testgrid/config/config.yaml]: Done.

**Comment by krzyzacy** [comment]
#5025 went in and now you need to rebase :-)

**Comment by michelle192837** [comment]
Looks good to me; will let @krzyzacy double-check.

**Comment by krzyzacy** [comment]
/lgtm
let's see

**Pull request merged**